bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -49,6 +50,23 @@ spec:
         - name: "mariadb-galera"
           image: {{ template "mariadb-galera.image" . }}
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          command:
+            - bash
+            - -ec
+            - |
+                # Bootstrap from the indicated node
+                HOSTNAME="$(hostname -s)"
+                if [[ "$HOSTNAME" =~ (.*)-([0-9]+)$ ]]; then
+                    NODE="${BASH_REMATCH[2]}"
+                    if [[ "$NODE" -eq "{{ int .Values.galera.bootstrap.bootstrapFromNode }}" ]]; then
```

Comment:
You can simplify this a lot:

- Add this to the "env" section:

```yaml
            - name: MY_POD_NAME
              valueFrom:
                fieldRef:
                  fieldPath: metadata.name
```

- Then you can obtain the node by simply doing:

```bash
          {{- $fullname := include "mariadb-galera.fullname" . }}
          {{- $bootstrapFromNode := int .Values.galera.bootstrap.bootstrapFromNode }}
          command:
            - bash
            - -ec
            - |
                # Bootstrap from the indicated node
                NODE_ID="${MY_POD_NAME#"{{ $fullname }}-"}"
                if [[ "$NODE_ID" -eq "{{ $bootstrapFromNode }}" ]]; then
```